packages/curl/build

```diff
@@ -4,6 +4,9 @@ export CPPFLAGS=-I/opt/mesosphere/include
 export LDFLAGS="-L/opt/mesosphere/lib -Wl,-rpath=/opt/mesosphere/lib"
 export LD_LIBRARY_PATH=/opt/mesosphere/lib
 
+# just so that first test does not yield failed directory check.
+mkdir -p /var/lib/dcos/pki/CA/trusted-certs
```

Comment:
fwiw, I'd prefer we stick with the "standard" directory layout -- is there any prior art for calling this directory trusted-certs?